lib/catalog/approval_transition.rb

```diff
@@ -10,9 +10,7 @@ def initialize(order_item_id)
     end
 
     def process
-      Insights::API::Common::Request.with_request(@order_item.context.transform_keys(&:to_sym)) do
```

Comment:
@bzwei Why has this code been removed, since this process is running async based on what is received from Kafka wouldn't we need this here. I know in the Kafka message payload we are sending the headers is that being set somewhere else now.

Reply:
@mkanoor Kafka listener switches context for every incoming message https://github.com/RedHatInsights/catalog-api/blob/master/lib/kafka_listener.rb#L41

Reply:
👍